progenitor-impl/src/method.rs

```diff
@@ -842,6 +887,13 @@ impl Generator {
                             ))
                         }
                     }
+                    OperationResponseType::Upgrade => {
+                        quote! {
+                            Err(Error::ErrorResponse(
+                                ResponseValue::upgrade(response).await?
+                            ))
+                        }
+                    }
```

Comment:
In what situation would we hit this?

Reply:
ah! my mistake, we wouldn't, as per below we don't yet describe any error responses in the generated OpenAPI schema.